pkg/services/pluginsintegration/config/config.go

```diff
@@ -38,6 +40,7 @@ func ProvideConfig(settingProvider setting.Provider, grafanaCfg *setting.Cfg) (*
 		grafanaCfg.PluginLogBackendRequests,
 		grafanaCfg.PluginsCDNURLTemplate,
 		tracingCfg,
+		featuremgmt.ProvidePluginsToggles(features),
```

Comment:
Related to other comment. Can't you just use `featuremgmt.ProvideToggles` here since `featuremgmt.FeatureToggles` interface is satisfied by `plugins.FeatureToggles`?